src/Symfony/Component/Config/Definition/Builder/EnumNodeDefinition.php

```
@@ -36,17 +37,28 @@ public function values(array $values): static
         return $this;
     }
 
+    public function enumFqcn(string $enumFqcn): static
+    {
+        if (!enum_exists($enumFqcn)) {
+            throw new \InvalidArgumentException(sprintf('The enum class "%s" does not exist.', $enumFqcn));
+        }
+
+        $this->enumFqcn = $enumFqcn;
+
+        return $this;
+    }
+
     /**
      * Instantiate a Node.
      *
      * @throws \RuntimeException
      */
     protected function instantiateNode(): EnumNode
     {
-        if (!isset($this->values)) {
-            throw new \RuntimeException('You must call ->values() on enum nodes.');
+        if (!isset($this->values) && !isset($this->enumFqcn)) {
+            throw new \RuntimeException('You must call either ->values() or ->enumFqcn() on enum nodes.');
```

should we allow calling _both_  ? What would be the effect ?

---

OK, apparently, right now, it silently ignores the `enumFqcn`